.github/workflows/release-alpha.yml

```diff
@@ -31,9 +31,9 @@ jobs:
 
       # Install pnpm because it is not included in our container image
       - name: install pnpm
-        uses: pnpm/action-setup@v2
+        uses: pnpm/action-setup@a7487c7e89a18df4991f7f222e4898a00d66ddda # v4
         with:
-          version: 8
+          version: 9
```

Comment:
why not 10?

Reply:
Returned to `pnpm@9` due to multiple issues with 10 and the CI